### ISSUE TITLE
The single pointer is not provided to adjust the column width of "Preview data" table. : A11y_SQL Azure Data Studio_Import Extension_Dragging Movements

### ISSUE THREAD

**Issue by Mana-swi** [microsoft/azuredatastudio#26428]
### Tags:
#A11yTCS;#A11yMAS;#WCAG2.5.7;#Dragging Movements;#A11ySev2;#Win32;#DesktopApp;#BM_SQL Azure Data Studio_Win32_Sep25;#SQL Azure Data Studio;#A11yWCAG2.2
**"[[Check out Accessibility Insights!]](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)) - Identify accessibility bugs before check-in and make bug fixing faster and easier."**
### Environment Details:
Application Name: SQL Azure Data Studio
Version: 1.52.0 (system setup)
VS Code: 1.82.0
Electron: 30.5.1
Chromium: 124.0.6367.243
Node.js: 20.16.0
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.22621
### Repro Steps:
1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Install the SQL Server Import extension.
4. Right click on any database under the connected server & enter Import Wizard.
5. Setp1 will appear. fill the required details and press enter on the "Next" button.
6. Setp2 will appear, tab till "preview data" table.
7. Observe that The single pointer is not provided to adjust the column width of "Preview data" table. 
### Actual:
The single pointer is not provided to adjust the column width of "Preview data" table. 

### Expected:
The single pointer should be provided to adjust the column width of "Preview data" table. 

### User Impact:
Not all users can accurately press and hold that contact while also repositioning the pointer. An alternative method must be provided so that users with mobility impairments who use a pointer (mouse, pen, or touch contact) can use the functionality.
### Recommendations:
 [MAS Problems and Suggested Fixes](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fmicrosoft.sharepoint.com%2fsites%2faccessibility%2fLists%2fMASProblemsAndSuggestedFixes%2fReviewed%2520Items.aspx&data=04%7c01%7cv-prajadhav%40microsoft.com%7cf642cb2fc53d4c4be38408d9d97cf216%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637779952946112692%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000&sdata=avswAOFlXEiWy%2fRT%2f%2fPGnurQzMGOUdSW7Z5Xp8ZioiQ%3D&reserved=0)
### MAS Reference:

- [MAS 2.5.7 - Dragging Movements](https://aka.ms/MAS2.5.7)

![Image](https://github.com/user-attachments/assets/a5505d8a-3401-42f3-a4d5-6ca08b09c006)
